test/parallel/test-http-server-headers-timeout-delayed-headers.js

```diff
@@ -53,10 +53,12 @@ server.listen(0, common.mustCall(() => {
 
   sendDelayedRequestHeaders = common.mustCall(() => {
     setTimeout(() => {
-      client.write('POST / HTTP/1.1\r\n');
-      client.write('Content-Length: 20\r\n');
-      client.write('Connection: close\r\n\r\n');
-      client.write('12345678901234567890\r\n\r\n');
+      client.write(
+        'POST / HTTP/1.1\r\n' +
+        'Content-Length: 20\r\n' +
+        'Connection: close\r\n\r\n' +
+        '12345678901234567890\r\n\r\n'
```

Comment:
For some of these it may actually make sense to retain multiple individual writes in the test just to exercise the streaming mechanisms more

Reply:
Yes, but I think it's outside of the scope of the test.